usaspending_api/disaster/management/sql/disaster_covid19_file_a.sql

```diff
@@ -0,0 +1,81 @@
+WITH latest_submissions AS (
+    SELECT
+        "dabs_submission_window_schedule"."submission_fiscal_year",
+        "dabs_submission_window_schedule"."is_quarter",
+        MAX("dabs_submission_window_schedule"."submission_fiscal_month") AS "submission_fiscal_month",
+        MAX("dabs_submission_window_schedule"."submission_reveal_date") AS "submission_reveal_date"
+    FROM "dabs_submission_window_schedule"
+    WHERE
+        "dabs_submission_window_schedule"."submission_reveal_date" <= now()
+    GROUP BY
+        "dabs_submission_window_schedule"."submission_fiscal_year",
+        "dabs_submission_window_schedule"."is_quarter"
+)
+
+SELECT
+    agency."name" AS "owning_agency_name",
+    CONCAT('FY', gtas."fiscal_year", 'P', gtas."fiscal_period") AS "submission_period",
```

Comment:
Minor thing.  All of these scripts are generating FY2020P8 instead of FY2020P08.  Everywhere else we're using two digits for monthly.

Reply:
Good catch. Pushed fix